docs/content/guides/author-apps/dapr/how-to-dapr-building-block/snippets/app-statestore.bicep

```diff
@@ -0,0 +1,130 @@
+import radius as radius
+
+@description('Specifies Kubernetes namespace for redis.')
+param namespace string = 'default'
+
+import kubernetes as kubernetes{
+  kubeConfig: ''
+  namespace: namespace
+}
+
+@description('The ID of your Radius Application. Automatically injected by the rad CLI.')
+param application string
+
+@description('The ID of your Radius environment. Set automatically by the rad CLI.')
+param environment string
+
+//CONTAINER
+resource demo 'Applications.Core/containers@2023-10-01-preview' = {
+  name: 'demo'
+  properties: {
+    application: application
+    container: {
+      image: 'ghcr.io/radius-project/samples/demo:latest'
+      ports: {
+        web: {
+          containerPort: 3000
+        }
+      }
+      livenessProbe: {
+        kind: 'httpGet'
+        containerPort: 3000
+        path: '/healthz'
+        initialDelaySeconds: 10
+      }
+    }
+    extensions: [
+      {
+        kind: 'daprSidecar'
+        appId: 'demo'
+        appPort: 3000
+      }
+    ]
+     connections: {
+      redis: {
+        source: stateStore.id
+      }
+    }
+  }
+}
+//CONTAINER
+
+//STATESTORE
+resource stateStore 'Applications.Dapr/stateStores@2023-10-01-preview' = {
```

Comment:
Please use a Recipe for this instead of manual resource provisioning.

Reply:
And add some text in the instructions noting that a recipe is being used, and link to the local-dev recipe to see the template being run.